packages/fether-electron/src/main/index.js

```diff
@@ -66,7 +96,75 @@ app.on('will-quit', killParity);
 
 app.on('quit', () => {
   pino.info('Leaving Fether');
+  app.releaseSingleInstanceLock();
   killParity();
 });
 
+// Security
+app.on('web-contents-created', (eventOuter, win) => {
+  win.on('will-navigate', (event, url) => {
+    // FIXME - check that parser is memory-safe
+    //
+    // Reference: https://letsencrypt.org/docs/certificates-for-localhost/
+
+    const parsedUrl = parseUrl(url);
```

Comment:
What do we need the `parse-url` library for? seems like we only use `parsedUrl.href` which should be about the same as `url` ?

Reply:
I used parseUrl as it was recommended here https://github.com/electron/electron/blob/master/docs/tutorial/security.md#how-10

Reply:
hmm there's no mention of the `parse-url` library in the link you gave. the snippet there is using `const URL = require('url').URL` (from nodejs) and is using so to get the "origin" part of the url

Reply:
Oh thanks, i've pushed a commit fixing this